src/ripple/rpc/handlers/ValidatorInfo.cpp

```diff
@@ -0,0 +1,46 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2019 Dev Null Productions
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <ripple/app/misc/ValidatorKeys.h>
+
+namespace ripple {
+Json::Value doValidatorInfo (RPC::Context& context)
+{
+    // return error if not configured as validator
+    if(context.app.validatorKeys().publicKey.empty())
+      return RPC::not_validator_error ();
+
+    Json::Value ret;
+
+    auto pk = context.app.validatorKeys().publicKey;
+    ret[jss::master_key]        = toBase58(TokenType::NodePublic, pk);
```

Comment:
Similarly, why not:

```
ret[jss::master_key] = toBase58(TokenType::NodePublic,
    context.app.getValidationPublicKey());
```

Also hate to complain about whitespace, but (here and elsewhere) I'd put one space before and one space after the `=` and avoid attempting to "align" the code manually.

Reply:
Done